### ISSUE TITLE
Could not run `agoo`

### ISSUE THREAD

**Issue by ioquatix** [ohler55/agoo#20]
```
koyoko% agoo
Traceback (most recent call last):
	4: from /home/samuel/.rvm/gems/ruby-2.5.0/bin/ruby_executable_hooks:15:in `<main>'
	3: from /home/samuel/.rvm/gems/ruby-2.5.0/bin/ruby_executable_hooks:15:in `eval'
	2: from /home/samuel/.rvm/gems/ruby-2.5.0/bin/agoo:23:in `<main>'
	1: from /home/samuel/.rvm/gems/ruby-2.5.0/bin/agoo:23:in `load'
/home/samuel/.rvm/gems/ruby-2.5.0/gems/agoo-2.1.1/bin/agoo:54:in `<top (required)>': undefined method `new' for Agoo::Server:Module (NoMethodError)
```

**Comment by ioquatix** [comment]
Also, how do you run agoo with a `config.ru` file?

**Comment by ohler55** [comment]
Reached out outside github. 
```
rackup -r agoo -s agoo
```

**Comment by ohler55** [comment]
Can this be closed?

**Comment by ioquatix** [comment]
It's up to you, I guess once you fix the bug, then you can close it?

**Comment by ohler55** [comment]
Ahh, I misunderstood. I thought you were asking how to run agoo. I see what you were referring to now.

**Comment by ohler55** [comment]
Fixed in release 2.2.2.
